src/Eryph.ClientRuntime/Eryph.ClientRuntime.csproj

```diff
@@ -2,35 +2,18 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <SignAssembly>false</SignAssembly>
-    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageProjectUrl>https://github.com/eryph-org/dotnet-clientruntime</PackageProjectUrl>
-    <Copyright>dbosoft GmbH</Copyright>
-    <Authors>dbosoft and Eryph contributors</Authors>
-    <Company>dbosoft GmbH</Company>
-    <Product>Eryph</Product>
-    <RepositoryUrl>https://github.com/eryph-org/dotnet-clientruntime</RepositoryUrl>
     <Description>Infrastructure for error handling, tracing, and HttpClient pipeline configuration. Required by Eryph client libraries generated using AutoRest.</Description>
-  </PropertyGroup>
-
-  <PropertyGroup>
     <Nullable>annotations</Nullable>
   </PropertyGroup>
 
   <PropertyGroup>
-    <LangVersion>12</LangVersion>
     <IncludeGeneratorSharedCode>true</IncludeGeneratorSharedCode>
     <RestoreAdditionalProjectSources>https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json</RestoreAdditionalProjectSources>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.39.0" />
-    <PackageReference Include="GitVersion.MsBuild" Version="5.12.0">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="System.Text.Json" Version="6.0.0" />
+    <PackageReference Include="Azure.Core" Version="1.44.1" />
+    <PackageReference Include="System.Text.Json" Version="8.0.5" />
   </ItemGroup>
 
   <ItemGroup>
```

Comment:
@fw2568, do you know why these ressources are present? Otherwise, I would remove them.